### PR TITLE
Added skip wallpaper flag

### PR DESCRIPTION
**Pull request by hallgat89**
This is a flag to skip setting the wallpaper. Why? Wal sets every wallpaper in "filled" mode, even if I would like to use a tiled pattern instead. With this flag I disable the wallpaper, and set the background myself.

### CODE REVIEW

File: wal
Comment (dylanaraps) [inline wal]:
```sh
if [[ -z "$nowall" ]]; then
```

File: wal
Comment (dylanaraps) [inline wal]:
```sh
out "wallpaper: '-n' was used, skipping wallpaper"
```

File: wal
Comment (dylanaraps) [inline wal]:
```sh
while getopts ":a:chi:no:qrt" opt; do
```

File: wal
Comment (dylanaraps) [inline wal]: Move this so that it's in alphabetical order with the other args.

**Review by dylanaraps — CHANGES_REQUESTED** [review]

**Review by hallgat89 — COMMENTED** [review]
It's okay for me :)

**Comment by dylanaraps** [comment]
It may work fine but it's not yet up to my standards. The PR only needs a few changes before it matches the style of the rest of the script.

I won't merge this until the above changes are made.

File: wal
Comment (hallgat89) [reply wal]: done

File: wal
Comment (hallgat89) [reply wal]: done

File: wal
Comment (hallgat89) [reply wal]: Done, and ordered

File: wal
Comment (hallgat89) [reply wal]: done and ordered

**Review by hallgat89 — COMMENTED** [review]
updated

File: wal
Comment (dylanaraps) [inline wal]: Remove this blank line.

File: wal
Comment (dylanaraps) [inline wal]: Remove this blank line.

**Review by dylanaraps — COMMENTED** [review]
Once this is done, I'll merge.

Thanks for the PR! :)

File: wal
Comment (hallgat89) [reply wal]: done

File: wal
Comment (hallgat89) [reply wal]: done

**Comment by dylanaraps** [comment]
LGTM :+1:

Thanks for the PR!

**Review by dylanaraps — APPROVED** [review]

**Comment by hallgat89** [comment]
Thanks for the cooperation :)

**Pull request merged**
